README.md

```diff
@@ -126,12 +126,25 @@ options in the [configuration file](#Configuration). All of these debug
 options are disabled by default. See the comments in the installed
 configuration file for further details.
 
-If you want to enable debug for all components, assuming a standard configuration file path, run:
+If you want to enable debug for all host-side components, assuming a standard configuration file path, run:
 
 ```bash
-$ sudo sed -i -e 's/^#\(enable_debug\).*=.*$/\1 = true/g' /usr/share/defaults/clear-containers/configuration.toml
+$ sudo sed -i -e 's/^#\(enable_debug\).*=.*$/\1 = true/g' -e 's/^kernel_params = ""/kernel_params = "agent.log=debug/g' /usr/share/defaults/clear-containers/configuration.toml
 ```
 
+The agent (`cc-agent`) which runs inside each virtual machine is slightly different: to enable its debug, set a
```

Comment:
The agent (`cc-agent`) that runs inside each virtual machine is slightly different. To enable its debug, set a